### DISCUSSION TITLE
Nothing happened after double clicking the webui.cmd file

### DISCUSSION THREAD

**Discussion by easonoob** [hlky/stable-diffusion-webui#1121] [Q&A]
Nothing happened after double clicking the `webui.cmd` file. I have followed all the instruction in the windows installation guide.

**Comment by easonoob** [comment]
It popped out a window when I open the file and after less than 0.1 seconds, it disappeared.

**Reply by raga-create** [reply]
Try running it with the cmd prompt so you have a chance to see the output from the webui.cmd launch. In my case it was solved by adding the "custom-conda-path.txt" to the main folder as described in the installation guide.

**Reply by easonoob** [reply]
Thanks, but how to add the custom-conda-path.txt ? I cannot find it in the installation guide. @raga-create

**Reply by raga-create** [reply]
"(Optional) Create a new text file in your root directory /stable-diffusion-webui/custom-conda-path.txt that contains the path to your relevant Miniconda3, for example C:\Users\<username>\miniconda3 (replace <username> with your own username). This is required if you have more than 1 miniconda installation or are using custom installation location."

**Reply by easonoob** [reply]
ok, thanks a lot!

**Reply by easonoob** [reply]
it works!
